tmpfiles.d/systemd.conf.m4

```diff
@@ -16,11 +16,6 @@ d /run/systemd/sessions 0755 root root -
 d /run/systemd/users 0755 root root -
 d /run/systemd/machines 0755 root root -
 d /run/systemd/shutdown 0755 root root -
-m4_ifdef(`ENABLE_NETWORKD',
-d /run/systemd/netif 0755 systemd-network systemd-network -
-d /run/systemd/netif/links 0755 systemd-network systemd-network -
-d /run/systemd/netif/leases 0755 systemd-network systemd-network -
-)m4_dnl
```

Comment:
i think this should be left in, so that regular userspace clients can install inotify watches on these dirs unconditionally whether networkd actually got started or didn't. After all, inotify is the way how event notifications work on the networkd client side...

Yes, strictly speaking these lines now become redundant, but only in the usual case, not in all cases